src/mono/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.xml

```diff
@@ -21,6 +21,9 @@
     <type fullname="System.Diagnostics.CodeAnalysis.SuppressMessageAttribute">
       <attribute internal="RemoveAttributeInstances" />
     </type>
+    <type fullname="System.Diagnostics.CodeAnalysis.RequiresAssemblyFilesAttribute">
```

Comment:
Does this need to be in the `<!-- Attributes that are allowed to be in any assembly -->` section?

https://github.com/dotnet/runtime/blob/38df267589c9a3359d27f47e83230d8aba93c598/src/mono/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.xml#L119-L132

I believe an assembly can just include an internal version of this assembly. For example:

https://github.com/dotnet/runtime/blob/38df267589c9a3359d27f47e83230d8aba93c598/src/libraries/Microsoft.Extensions.DependencyModel/src/Microsoft.Extensions.DependencyModel.csproj#L16

Reply:
Moved it together with SuppressMessageAttribute